content/de/blog/v2-release/index.md

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Datenanfragen.de v2: Nagelneue Oberfläche und Beschwerdegenerator",
+    "type": "blog",
+    "date": "2022-12-07T12:00:00+02:00",
+    "description": "Gestern haben wir die neue Version der Webseite mit vielen neuen Features veröffentlicht: Eine neue Oberfläche für den Anfragengenerator, eine umfangreichere Anfragenhistorie, eine schicke neue Startseite und ein Generator für Beschwerden und Mahnungen.",
+    "featured_image": "stockings-feature",
```

Comment:
The background needs to be a lot darker and bluer for consistency and color contrast.